vagrant/setup.sh

```diff
@@ -30,6 +30,24 @@ Argument:
 EOF
 }
 
+function enable_iommu {
+    echo "Checking Intel iommu"
+    if ! find /sys/kernel/iommu_groups/*/devices/* > /dev/null 2>&1; then
```

Comment:
we use `/sys/class/iommu` elsewhere (e.g., in NFD).

Reply:
This script was designed to be used enabling QAT needs.

Reply:
I'm not sure I follow. I just suggested an alternate path to check whether IOMMU is enabled.

Reply:
@mythi is that alternate path is not based on the distro used?

Reply:
that path is the kernel interface to check IOMMU is present